packages/core/src/App/Containers/__tests__/trade-notifications.spec.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { SwipeableNotification } from '@deriv/components';
+import { StoreProvider, mockStore, useStore } from '@deriv/stores';
+import { getCardLabels } from '@deriv/shared';
+import TradeNotifications from '../trade-notifications';
+
+type TMockedSwipeableNotificationProps = React.ComponentProps<typeof SwipeableNotification> & {
+    onUnmount?: () => void;
+    timestamp?: number | null;
+};
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    MobileWrapper: jest.fn(({ children }) => children),
+    SwipeableNotification: ({ children, onUnmount, timestamp }: TMockedSwipeableNotificationProps) => {
+        if (onUnmount) onUnmount();
+        return (
+            <div>
+                {children}
+                {!!timestamp && <div>Timestamp</div>}
```

Comment:
Hmmm, why not  `{timestamp && <div>Timestamp</div>}` ?

Reply:
@kate-deriv just because timestamp can be a number :) e.g. if I tested it with 0, 0 will be rendered.